babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,15 +788,71 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList paramNames = new SqlNodeList(getPos());
+    SqlNodeList paramValues = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [
+        SqlExecMacroArgument(paramNames, paramValues)
+    ]
     {
-        return new SqlExecMacro(s.end(this), macro);
+        return new SqlExecMacro(s.end(this), macro, paramNames, paramValues);
     }
 }
 
+void SqlExecMacroArgument(SqlNodeList paramNames, SqlNodeList paramValues) :
+{
+    SqlNode name;
+    SqlNode value;
+}
+{
+        <LPAREN>
+        (
+            value = Literal()
+            {
+                paramValues.add(value);
+            }
+
+        )
+        (
+            <COMMA>
+            value = Expression(ExprContext.ACCEPT_SUB_QUERY)
+            {
+                paramValues.add(value);
+            }
+
+        )*
+        <RPAREN>
+    |
+        <LPAREN>
+        (
+            name = SimpleIdentifier()
+            <EQ>
+            {
+                paramNames.add(name);
+            }
+            value = Expression(ExprContext.ACCEPT_SUB_QUERY)
+            {
+                paramValues.add(value);
+            }
+        )
```

Comment:
No need for parentheses (lines 830 & 840)

Reply:
Removed parentheses in https://github.com/googleinterns/calcite/pull/108/commits/41eff94023a11dbee3d010b4db619eae40217431